src/applications/edu-benefits/tests/0994/pages/bankInformation.unit.spec.js

```diff
@@ -41,12 +37,7 @@ describe('Bank Information', () => {
     form.unmount();
   });
 
-  it('successfully submits ', () => {
-    const {
-      accountType,
-      accountNumber,
-      routingNumber,
-    } = initialData.bankAccount;
+  it('successfully submits with prefill data ', () => {
```

Comment:
can you add "successfully submits without prefill data" back in so we don't lose it?

Reply:
It was never in there as a successful submit without prefill data. It checked to make sure it failed submission